Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -1313,24 +1313,66 @@ void MenuBar::GenerateSymbolsFromRSOAuto()
 
       if (!found_addr.has_value())
         break;
+
       next = *found_addr + 1;
 
-      // Get the beginning of the string
-      found_addr = accessors->Search(*found_addr, reinterpret_cast<const u8*>(""), 1, false);
-      if (!found_addr.has_value())
-        continue;
+      // Non-null data can precede the module name.
+      // Get the maximum name length that a module could have.
+      auto get_max_module_name_len = [found_addr] {
+        u32 len = 0;
+
+        while (true)
```

Comment:
The infinite loop here makes me a bit uncomfortable -- do we have an upper bound on the maximum length?

Reply:
The only limit that I can think of is the max value of a `uint` as that's the type of `module_name_length`. But, we could give a upper limit of `MAX_PATH`. Thoughts?

Reply:
Assuming that most module names are Windows paths, you could create a constant similar to `MAX_PATH` on Windows with a limit of 260 characters, I guess.

Reply:
Using MAX_PATH=260 sounds fine to me.

Reply:
I opted to name the constant `MODULE_NAME_MAX_LENGTH`, to avoid collision with the Windows's `MAX_PATH` constant